build.go

```diff
@@ -134,12 +136,18 @@ func (vs *version) addFile(bs *blocks.BlockStore, file string) error {
 		return fmt.Errorf("reading %s: %s", disp, err)
 	}
 
-	sf := sequencefile.New(stream)
+	sf := sequencefile.NewReader(bufio.NewReader(stream))
```

Comment:
Yeah, the `sequencefile` in this case changed from the internal library to the external one (in the import statement) and the interface is slightly different.